src/r5py/util/memory_footprint.py

```diff
@@ -124,20 +124,20 @@ def _get_max_memory(max_memory):
     else:
         # convert to MiB
         if unit is None:
-            value *= 2**-20
+            value = value >> 20
         elif unit == "K":
-            value *= 2**-10
+            value = value >> 10
         elif unit == "M":
-            value *= 2**1
+            value = int(value * 1.024)
```

Comment:
What does this do? Is it converting from Megabytes to Mibibytes?

Reply:
Yes it converts Megabytes to Mibibytes. The original logic returned incorrect result.